bin/gwdetchar-channel-diff

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# coding=utf-8
+# Copyright (C) LIGO Scientific Collaboration (2017-)
+#
+# This file is part of the GW DetChar python package.
+#
+# GW DetChar is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# GW DetChar is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GW DetChar.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Find settings that change between two times
+"""
+
+from __future__ import print_function
+
+from gwpy.io import datafind
+from gwpy.io.gwf import get_channel_names
+from gwpy.timeseries import TimeSeriesDict
+from numpy import diff
+
+def find_sec_frames(ifo,start,end):
+    ifo_lett = ifo[0]
+    conn = datafind.connect()
+    cache = conn.find_frame_urls(ifo_lett, ifo_lett+'1_T',
+                                start, end, urltype='file')
+    return cache
+
+import argparse
+parser = argparse.ArgumentParser(description='Find state changes.')
+parser.add_argument('-i','--ifo', type=str, help='IFO', required=True)
+parser.add_argument('-s','--t1', type=int, help='First GPS time', required=True)
+parser.add_argument('-e','--t2', type=int, help='Second GPS time', required=True)
```

Comment:
@andrew-lundgren, is there a reason to use `required` positional arguments, rather than just options? The `gwdetchar.cli` module provides some default options that you can use, e.g.

```python
from gwdetchar import cli
parser = cli.create_parser(description=__doc__)
cli.add_gps_start_stop_arguments(parser)
cli.add_ifo_option(parser)
```

[snippet from [`gwdetchar-software-saturations`](https://github.com/ligovirgo/gwdetchar/blob/d7a356f792e64c7f568f3cbdb9c10b3a29243072/bin/gwdetchar-software-saturations#L257)]

This then allows users to call 

```bash
gwdetchar-sofware-saturations gpsstart gpsend --ifo X1
```